core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogInitialization.java

```diff
@@ -528,8 +528,60 @@ private void startBundle(OsgiBundleInstallationResult br) throws BundleException
         }
     }
 
+    private PersistedCatalogState filterPersistedState(PersistedCatalogState persistedState, RebindLogger rebindLogger) {
```

Comment:
rename `filterBundlesAndCatalogInPersistedState` to clarify this isn't doing any other filtering or transforming instances